### PR TITLE
CPDP-2752: [AR] Individual start page changes

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html [hunk 1]
```diff
@@ -98,16 +98,6 @@ <h2 class="heading-medium" id="howTo">@messages("landingPage.howTo.h2")</h2>
         <li>@messages("landingPage.howTo.list1.bullet3")</li>
     </ul>
 
-    <div class="panel panel-border-wide">
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:101]: this needs to be feature flagged - if amend returns is enabled we want this and if it's disabled we want the old content. There already exists a flag you can access via ViewConfig called `amendReturnsEnabled` or something

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html [hunk 2]
```diff
@@ -98,15 +98,17 @@ <h2 class="heading-medium" id="howTo">@messages("landingPage.howTo.h2")</h2>
...
-        <p>
-            @if(appConfig.furtherReturnsEnabled){
+    @if(!appConfig.amendReturnsEnabled){
+        <div class="panel panel-border-wide">
+            <p>
+                @if(appConfig.furtherReturnsEnabled){
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:104]: can you ever reach this statement with `!appConfig.amendReturnsEnabled` above?

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html [hunk 3]
```diff
@@ -95,18 +95,28 @@ <h2 class="heading-medium" id="howTo">@messages("landingPage.howTo.h2")</h2>
...
+                @messages("landingPage.howTo.list1.bullet3")
+            }
+        </li>
     </ul>
 
-    <div class="panel panel-border-wide">
-        <p>
-            @if(appConfig.furtherReturnsEnabled){
+    @if(appConfig.amendReturnsEnabled){
+    }else{
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:109]: you don't need an `if-else` here - couldn't you just have `if(!appConfig.amendReturnsEnabled))`?

Comment (rambabu-posa) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:109]: Are you looking for this please?
```
    @if(!appConfig.amendReturnsEnabled){
        <div class="panel panel-border-wide">
            <p>
                @if(appConfig.furtherReturnsEnabled){
                @Html(messages("landingPage.howTo.furtherReturns", appConfig.contactHmrc))
                } else{
                @Html(messages("landingPage.howTo.firstReturnsOnly", appConfig.contactHmrc))
                }
            </p>
        </div>
    }
```

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:109]: yeah, there's no point having an empty branch in an if-else statement

Comment (rambabu-posa) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html:109]: Sorry Initially I did same thing and confused with your 2nd comment
`can you ever reach this statement with !appConfig.amendReturnsEnabled above?
`

**Pull request merged**